GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Set and Get GVFS config settings.")]
+    public class ConfigVerb : GVFSVerb
+    {
+        private const string ConfigVerbName = "config";
+        private LocalGVFSConfig localConfig;
+
+        public override string EnlistmentRootPathParameter { get; set; }

```

Comment:
Throw if this property is accessed (see `ServiceVerb` as an example). This command is not tied to any particular repo.